cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -296,10 +366,39 @@ class Home extends Component {
     );
 
     let entitiesToBeRendered;
+    let bodyContent;
+
     if(this.state.loading){
       entitiesToBeRendered = loading;
-    } else if(this.state.entities.length === 0) {
+
+      bodyContent = () => {
+          return (
+            <div className="entity-list">
+                <div className="entities-container">
+                  <ReactCSSTransitionGroup
```

Comment:
`ReactCSSTransitionGroup` without any transitionName? Will the animation still work?